SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.h

```diff
@@ -2,17 +2,21 @@
 
 #pragma once
 
-#include "Tests/TestDefinitions.h"
+#include "Misc/AutomationTest.h"
```

Comment:
This change reflects what this file _really_ wanted to import, without a middle-man import 
(Fortunately, this resolved a circular dependency)